pkg/shed/field_string.go

```diff
@@ -49,7 +49,7 @@ func (db *DB) NewStringField(name string, logger logging.Logger) (f StringField,
 func (f StringField) Get() (val string, err error) {
 	b, err := f.db.Get(f.key)
 	if err != nil {
-		if err == leveldb.ErrNotFound {
+		if err == ErrNotFound {
 			f.logger.Errorf("key %s not found", string(f.key))
```

Comment:
if there's an error, you would would probably want to return it no? here you log it, but then return an empty string but nil error. i'm not sure this is right

Reply:
ok. The idea was to split the error in to KeyNotFound and rest. because the application might want to respond differently... but since you guys seems to like the idea of just returning the err as it is, i am fixing in all those places.